examples/shopify.rb

```diff
@@ -0,0 +1,71 @@
+class Shopify
+  attr_reader :client
+
+  def initialize(account)
+    @client = account
+  end
+
+  def shop
+    ShopifyAPI::Session.temp(@client.uid, @client.access_token){ ShopifyAPI::Shop.current }
```

Comment:
Will the Shop be valid outside the temp call? It seems like this should accept a block and run the result in that context instead.

Reply:
@drewish 

not it will not for that you have to create a session and thuen create obj and use them like 
`session = ShopifyAPI::Session.new(@client.uid, @client.access_token)
ShopifyAPI::Base.activate_session(session)

shop = ShopifyAPI::Shop.current`